src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputFormattedEmptyFinallyBlocks.java

```diff
@@ -0,0 +1,100 @@
+///////////////////////////////////////////////////////////////////////////////////////////////
+// Test case file for checkstyle.
+// Created: 2001
+///////////////////////////////////////////////////////////////////////////////////////////////
+
+package com.google.checkstyle.test.chapter4formatting.rule413emptyblocks;
+
+class InputFormattedEmptyFinallyBlocks {
+  boolean flag;
+
+  void doSm() {}
+
+  void foo() {
+    try {
+      if (!flag) {
+        doSm();
+      }
+    } catch (Exception e) {
+      /* ignore */
+    } finally { // ok
+      /* ignore */
+    } // ok
+  }
+
+  void foo2() {
+    try {
+      if (!flag) {
+        doSm();
+      }
+      // violation below 'Empty catch block.'
+    } catch (Exception e) {
+    } finally {
+    } // violation above 'Empty finally block.'
+  }
+
+  class Inner {
+    boolean flag;
+
+    void doSm() {}
+
+    void foo() {
+      try {
+        if (!flag) {
+          doSm();
+        }
+      } catch (Exception e) {
+        /* ignore */
+      } finally { // ok
+        /* ignore */
+      } // ok
+    }
+
+    void foo2() {
+      try {
+        if (!flag) {
+          doSm();
+        }
+        // violation below 'Empty catch block.'
+      } catch (Exception e) {
+      } finally {
+      } // violation above 'Empty finally block.'
+    }
+  }
+
+  Inner anon =
+      new Inner() {
+        boolean flag;
+
+        void doSm() {}
+
+        void foo() {
+          try {
+            if (!flag) {
+              doSm();
+            }
+          } catch (Exception e) {
+            /* ignore */
+          } finally { // ok
+            /* ignore */
+          } // ok
+        }
+
+        void foo2() {
+          try {
+            if (!flag) {
+              doSm();
+            }
+            // violation below 'Empty catch block.'
+          } catch (Exception e) {
+          } finally {
+          } // violation above 'Empty finally block.'
+        }
+      };
+
+  interface InterfaceEndingWithSemiColon2 {
+    public void fooEmpty();
+  }
+  ; // ok until #7541
+  // violation above '';' should be separated from previous line.'
```

Comment:
not sure if this is a false positive or negative

Reply:
>   ; // ok until #7541
>   // violation above '';' should be separated from previous line.'

I don't understand why we have it here. The input is to cover empty `finally` blocks that is not relevant to the aforementioned lines. @Zopsss could you remove them?

Reply:
done